float64 velocity
				# 100 = edge_case(must be handled first);
				# 99 - 51 = less than edge, more than traffic;	
				# 49 - 2 = less than traffic, more than traj.
int8 priority
int8 node_id